### ISSUE TITLE
Swagger output is followed by Welcome Page

### ISSUE THREAD

**Issue by colemickens** [domaindrivendev/Ahoy#3]
Hello!

I'm using Ahoy with a couple modifications against beta5. I say that so that you can potentially ignore this bug report since beta5 is still under development.

Anyway, when I hit my Swagger endpoint, I see the swagger json, but it's immediately followed by the ASP.NET Welcome Page content (in the same response).

```
app.UseSwagger("api/{apiVersion}/swagger.json");

app.UseStatusCodePages();
app.UseErrorPage();
app.UseCors(policy => policy.WithOrigins("http://polykube.io"));
app.UseMvc();
app.UseWelcomePage();
```

Commenting out the last line allows me to consume the swagger json as I'd expect. Not sure if I've done something wrong, if I'm using it incorrectly, or if it's simply a bug in ASP.NET5.

**Comment by rynowak** [comment]
See: https://github.com/domaindrivendev/Ahoy/blob/master/src/Swashbuckle/Application/SwaggerDocsHandler.cs#L32

The route handler needs to set `context.IsHandled` to true to stop processing.

**Comment by colemickens** [comment]
Yes, that did the trick. Thanks @rynowak.

**Comment by domaindrivendev** [comment]
Fixed in latest release - Swashbuckle 6.0.0-beta2
